notebooks/COS/AsnFile/AsnFile.ipynb

```diff
@@ -0,0 +1,578 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"topAF\"></a>\n",
+    "\n",
+    "# Modifying or Creating an Association File\n",
+    "\n",
+    "# Learning Goals\n",
+    "<font size=\"4 \">This Notebook is designed to walk the user (*you*) through: **Creating or altering the association (`asn`) file used by the Cosmic Origins Spectrograph (*COS*) pipeline to determine which data to process**:\n",
+    "    \n",
+    "<font size=\"3\"> 1. [**Examining an association file**](#examAF)\n",
+    "\n",
+    "<font size=\"3\"> 2. [**Editing an existing association file**](#editAF)\n",
+    "\n",
+    "\\- 2.1. [Removing an exposure](#subAF)\n",
+    "    \n",
+    "\\- 2.2. [Adding an exposure](#addAF)\n",
+    "    \n",
+    "<font size=\"3\"> 3. [**Creating an entirely new association file**](#newAF)\n",
+    "\n",
+    "\\- 3.1. [Simplest method](#simpleAF)\n",
+    "    \n",
+    "\\- 3.2. [With fits header metadata](#metaAF)\n",
+    "    \n",
+    "<!--    #### 3. [**Reprocessing the data**](#calcosAF) **Optional** -->"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 0. Introduction\n",
+    "**The Cosmic Origins Spectrograph ([*COS*](https://www.nasa.gov/content/hubble-space-telescope-cosmic-origins-spectrograph)) is an ultraviolet spectrograph on-board the Hubble Space Telescope ([*HST*](https://www.stsci.edu/hst/about)) with capabilities in the near ultraviolet (*NUV*) and far ultraviolet (*FUV*).**\n",
+    "\n",
+    "**This tutorial aims to prepare you to alter the association file used by the `calcos` pipeline.** Association files are `fits` files containing a binary table extension, which lists science and calibration exposures which the pipeline will process together.\n",
+    "\n",
+    "- For an in-depth manual to working with COS data and a discussion of caveats and user tips, see the [COS Data Handbook](https://hst-docs.stsci.edu/display/COSDHB/).\n",
+    "- For a detailed overview of the COS instrument, see the [COS Instrument Handbook](https://hst-docs.stsci.edu/display/COSIHB/).\n",
+    "\n",
+    "<font size=\"4 \">We'll demonstrate creating an `asn` file in two ways:\n",
+    "1. **Editing an existing `asn` file to add or remove an exposure**\n",
+    "2. **Creating an entirely new `asn` file**"
```

Comment:
Somewhat qualitative, but I'd suggest you change this from a "numbered list" to just a regular prose sentence.  The numbered list renders a bit odd for me,  and I think it sort of distracts from what you're saying to make it a "numbered list".

Also I think you need a `</font>` at the end of where you want the font size?  It may not be absolutely necessary, but I think it's compatible with more browsers that way.